validator/project_validator.go

```diff
@@ -982,6 +983,20 @@ func verifyTaskRequirements(project *model.Project) ValidationErrors {
 	return errs
 }
 
+func validateTaskPatchability(project *model.Project) ValidationErrors {
```

Comment:
Should this also verify GitTagOnly isn't set with patchable/patch only?

Reply:
Are tasks with git tag only implicitly unpatchable?

Reply:
I added a check to ensure that `git_tag_only: true` isn't usable with `patchable: true` or `patch_only: true`.